distribution/docs/src/main/resources/_contents/_tables/NSILI_Connected_Source-table-contents.adoc

```diff
@@ -0,0 +1,222 @@
+.[[NSILI_Federated_Source]]NSILI Federated Source
+[cols="1,1m,1,3,1,1" options="header"]
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|NSILI Source ID
+|id
+|String
+|The ID of the NSILI Source
+|NSILI Federated Source
+|true
+
+| Ior File URL
+| iorUrl
+| String
+| The URL of the IOR file to use for the NSILI CORBA Service. Supported URL protocols are 'http://'
+| null
+| true
+
+| HTTP/S or FTP Username
+| serverUsername
+| String
+| 'https://'
```

Comment:
Looks like something went wrong when transferring this over? Same for `HTTP/S or FTP Password` and `HTTP/S or FTP Timeout` descriptions.